Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -718,6 +729,23 @@ def addOrthogonalSeeding(
     return seedingAlg.config.outputSeeds
 
 
+def addHoughTransformSeeding(
+    sequence: acts.examples.Sequencer,
+    config: acts.examples.HoughTransformSeeder.Config = acts.examples.HoughTransformSeeder.Config(),
```

Comment:
```suggestion
    config: acts.examples.HoughTransformSeeder.Config,
```

I think we dont need to repeat the default here